src/App.tsx

```diff
@@ -16,10 +17,10 @@ const App: FunctionComponent = () => {
             <Home />
           </Route>
           <Route path="/beacon-records">
-            <BeaconRecords />
+            <BeaconRecords beaconsGateway={beaconsGateway} />
           </Route>
         </Switch>
-        <Footer />
+        {/*<Footer />*/}
```

Comment:
Footer currently displaying on top of page content, not below it.  Working on a fix to replace.